docs/topics/migrations.txt

```diff
@@ -715,6 +715,12 @@ You must then transition the squashed migration to a normal migration by:
     Once you've squashed a migration, you should not then re-squash that squashed
     migration until you have fully transitioned it to a normal migration.
 
+.. versionchanged:: 4.0
+
+    Support for migrating to replaced migrations still present on disk was
+    added.
+    In this scenario, no squashed migration will be executed: only the
+    migrations they originally replaced, even for dependencies in other apps.
```

Comment:
It's probably due to me not being that familiar with migrations, but I'm really struggling with what this means. I wonder if a small example could help?

Reply:
Yup, it would be better to avoid the term "squashed migration" -- which is the one that's squashed: the one that _got_ squashed or the one that _did_ the squashing? (The one that did the squashing gets labeled "...squashed..."!)

Open to the idea of a small example but don't want to go there just quite yet until I'm sure this change is acceptable. This doc has other problems, for instance: earlier it says it's okay to commingle replaced/replacing migrations, then below it says you are strongly encouraged to get rid of the old ones, and when doing so you "must" do the following to convert the "squashed" migration to "regular" etc. I'm just not sure this is the PR for a big rewrite. Speaking of which, we have #14408.

Reply:
Thankfully, this has been mooted by Mariusz's suggested smaller diff!